c/busybox-1.22.0-todo/cal_false-unreach-call_true-no-overflow_true-valid-memsafety.i

```diff
@@ -3790,18 +3790,22 @@ int main()
   int argc = __VERIFIER_nondet_int();
   __VERIFIER_assume(argc >= 1 && argc <= 10000);
   char **argv=malloc((argc+1)*sizeof(char*));
+  char **mem_track=malloc((argc+1)*sizeof(char*));
+  char **mem_argv=argv;
```

Comment:
I don't think that copy is necessary, the pointer as such is passed by value and cannot possibly be changed.

Reply:
Yes, I agree. I missed that before.